.github/workflows/ci.yml

```diff
@@ -265,12 +238,36 @@ jobs:
         with:
           submodules: recursive
 
-      - name: Build godot-cpp
-        run: |
-          cmake -DCMAKE_BUILD_TYPE=Release -G"Visual Studio 16 2019" .
-          cmake --build . --verbose --config Release
-
       - name: Build test GDExtension library
         run: |
-          cd test && cmake -DCMAKE_BUILD_TYPE=Release -DGODOT_HEADERS_PATH="../godot-headers" -DCPP_BINDINGS_PATH=".." -G"Visual Studio 16 2019" .
-          cmake --build . --verbose --config Release
+          mkdir cmake-build
+          cd cmake-build
+          cmake ../
+          cmake --build . --verbose -t godot-cpp-test --config Release
+
+#      - name: Download latest Godot artifacts
+#        uses: dsnopek/action-download-artifact@1322f74e2dac9feed2ee76a32d9ae1ca3b4cf4e9
+#        with:
+#          repo: godotengine/godot
+#          branch: master
+#          event: push
+#          workflow: windows_builds.yml
+#          workflow_conclusion: success
+#          name: windows-x86_64-msvc
+#          search_artifacts: true
+#          check_artifacts: true
+#          ensure_latest: true
+#          path: godot-artifacts
+
+#          FIXME Error: no matching workflow run found with any artifacts?
+
+#      - name: Run tests
+#        run: |
+#          # Run the test project
+#          $godot="./godot-artifacts/godot.windows.editor.x86_64.exe"
+#          &$godot -e --path test\project\ --headless --quit *>$null
+#          $godot_tr = C:\build\godot\msvc.master\bin\godot.windows.template_release.x86_64.exe
+#          $godot_tr --path ..\test\project\ --headless --quit
+#          $output=&$godot --path test\project\ --headless --quit *>&1 | Out-String
+#          !$output.Contains("==== TESTS FINISHED ====")
+#          $output.Contains("******** FAILED ********")
```

Comment:
This commented out code will need to be removed (or made to work?) before this PR will be mergeable

Reply:
Commented code has been removed